### PR TITLE
Updates to get-involved page

### PR DESCRIPTION
**Pull request by shredtechular**
This pull request includes updates that address #11.

The following changes were made to address the associated issue:
- updated the owl images next to the 3 CTA buttons
- added text underneath the Volunteer section
- added thumbnail images of the clothing requested in the issue
- addressed some simple mobile styling issues

### CODE REVIEW

**Comment by shredtechular** [comment]
@j00bar this pull request is ready for review

File: get-involved.html [hunk 1]
```diff
@@ -130,19 +138,29 @@
...
 							</div>
 						</div>
+						<div class="col-xs-12 text-center volunteers">
+							<span class="subheader">CURRENTLY RECRUITING VOLUNTEERS WITH THE FOLLOWING SKILLS:</span>
+							<ul class="positions-list">
+								<li>Fundraising</li>
+								<li>Press relations</li>
+								<li>Accounting work</li>
+								<li>Grant writing</li>
+								<li>SEO & social media marketing</li>
```
Comment (j00bar) [inline get-involved.html]: Do you mind converting this to an `&amp;`? Thanks!

Comment (shredtechular) [reply get-involved.html]: HTML Entity used now, fixed in 9030390

@j00bar ready for round 2 of review.

**Comment by j00bar** [comment]
Sorry - conflicts.

**Comment by shredtechular** [comment]
@j00bar rebased this feature branch with `master` and resolved conflicts.  This is ready for your review again.  

**NOTE** - depending on the order these pull requests are merged, we could see conflicts again.

**Pull request merged**